docs/adr/index.md

```diff
@@ -0,0 +1,75 @@
+# Architectural Decision Records
+
+This is a location to record all architecture decisions in the Oasis Core
+project via [Architectural Decision Records] (ADRs).
+
+<!-- markdownlint-disable line-length -->
+[Architectural Decision Records]: https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions.html
+<!-- markdownlint-enable line-length -->
+
+## Format
+
+Each record has a unique number associated with it to make it easier to
+cross-reference them. The records are stored as Markdown files in this
+directory, named using the following convention:
+
+```
+<adr-number>-<short-title>.md
+```
+
+Where:
+
+* `<adr-number>` is the assigned zero-padded ADR number.
+* `<short-title>` is the ADR's title in [Kebab case].
+
+The content of each ADR should follow [the template]. In short, an ADR should
+provide:
+
+* a changelog of all modifications so far,
+* context on the relevant goals and the current state,
+* proposed changes to achieve the goals,
+* summary of pros and cons and
```

Comment:
Perhaps also explicitly state that an ADR should also include the decision:

```suggestion
* proposed changes to achieve the goals,
* decision was made,
* summary of pros and cons and
```

Reply:
Good idea, done.